src/features/settings/gestures/SwipeSettings.tsx

```diff
@@ -0,0 +1,233 @@
+import { IonActionSheet, IonLabel, IonList, IonToggle } from "@ionic/react";
+import { InsetIonItem, ListHeader } from "../shared/formatting";
+import { useAppDispatch, useAppSelector } from "../../../store";
+import {
+  OSwipeActionPost,
+  OSwipeActionComment,
+  SwipeAction,
+  SwipeActions,
+  OSwipeActionInbox,
+  OSwipeActionAll,
+} from "../../../services/db";
+import SettingSelector from "../shared/SettingSelector";
+import {
+  setPostSwipeActionFarEnd,
+  setPostSwipeActionFarStart,
+  setPostSwipeActionEnd,
+  setPostSwipeActionStart,
+  setCommentSwipeActionFarEnd,
+  setCommentSwipeActionFarStart,
+  setCommentSwipeActionEnd,
+  setCommentSwipeActionStart,
+  setInboxSwipeActionFarEnd,
+  setInboxSwipeActionFarStart,
+  setInboxSwipeActionEnd,
+  setInboxSwipeActionStart,
+  setAllSwipesToDefault,
+  setDisableLeftSwipes,
+  setDisableRightSwipes,
+} from "./gestureSlice";
+import { ActionCreatorWithPayload } from "@reduxjs/toolkit";
+import { Dictionary } from "lodash";
+import { useState } from "react";
+import { IonActionSheetCustomEvent, OverlayEventDetail } from "@ionic/core";
+import { ReactComponent as ShortSwipeSvg } from "./swipe_short.svg";
```

Comment:
I just realized that these files have underscores instead of camelCase too :)

Reply:
This was to follow the naming in the public folder, it seemed like svg files used snake case.

Reply:
Oh, that's not right. We should be doing camel case (title case for react components).

Reply:
I can fix it tomorrow or feel free to push a fix